app/views/nsm/steps/youth_court_claim_additional_fee/edit.html.erb

```diff
@@ -0,0 +1,19 @@
+<% title t('.page_title') %>
+<% decision_step_header %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= govuk_error_summary(@form_object) %>
+    <h1 class="govuk-heading-l"><%= t('.heading') %></h1>
+    <h2 class="govuk-hint govuk-!-margin-bottom-6"><%= t '.subheading' %></h2>
+
+    <%= step_form @form_object do |f| %>
+      <%= f.govuk_radio_buttons_fieldset :youth_court_fee_claimed, legend: nil do %>
+        <%= f.govuk_radio_button :youth_court_fee_claimed, YesNoAnswer::YES.to_s, label: { text: t('helpers.label.nsm_steps_youth_court_claim_additional_fee_form.options.yes' ) } %>
```

Comment:
If you have multiple radio buttons, you need to add `link_errors: true` to the first one so that if there's a validation error, the summary message will link to the form element.

You might be better off using `govuk_collection_radio_buttons` which will handle that sort of thing for you.